src/maquette.js

```diff
@@ -126,12 +126,12 @@
         }
       } else if(propName === "key") {
         continue;
-      } else if(propValue === null || propValue === undefined) {
+      } else if(propValue == null) {
         continue;
       } else {
         var type = typeof propValue;
         if(type === "function") {
-          if(eventHandlerInterceptor && (propName.lastIndexOf("on", 0) === 0)) { // lastIndexOf(,0)===0 -> startsWith
```

Comment:
lastIndexOf(,0) is used on purpose here, it does not scan the whole string if the first character does not match.